nx-X11/programs/Xserver/Xext/Imakefile

```diff
@@ -79,7 +79,7 @@ XF86BIGFOBJS = xf86bigfont.o
 
        SRCS = shape.c $(SHMSRCS) xcmisc.c\
               xtest.c xtest1di.c xtest1dd.c sleepuntil.c \
-	      bigreq.c sync.c $(SCRNSAVSRC) \
+	      bigreq.c sync.c syncsrv.h $(SCRNSAVSRC) \
```

Comment:
Don't list header (syncsrv.h) files in SRCS variables. Only to-be-compiled .c files should be listed under SRCS.